packages/eslint/src/generators/init/init.ts

```diff
@@ -49,6 +50,21 @@ function addTargetDefaults(tree: Tree) {
   updateNxJson(tree, nxJson);
 }
 
+export function updateVsCodeRecommendedExtensions(host: Tree) {
```

Comment:
With the suggested change in the test case, we don't need to export this function anymore.
```suggestion
function updateVsCodeRecommendedExtensions(host: Tree) {
```